package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@joinbox/loopback-nunjucks",
+  "version": "0.0.1",
+  "description": "Nunjucks integration component for Loopback 3.x.x applications with simple asset management. ",
+  "main": "index.js",
+  "scripts": {
+    "lint": "node_modules/.bin/eslint .",
+    "test:base": "mocha $NODE_DEBUG_OPTION $WATCH_TESTS --recursive --check-leaks --full-trace --exit --timeout 12000",
+    "test:unit": "npm run test:base ./test/support/setup-unit.js ./test/unit",
+    "test:integration": "npm run test:base ./test/support/setup-integration.js ./test/integration",
+    "test": "npm run test:base ./test/support/setup-integration.js ./test/unit ./test/integration",
+    "test:coverage": "nyc npm test"
+
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/joinbox/loopback-nunjucks.git"
+  },
+  "keywords": [
+    "Loopback",
+    "Nunjucks"
+  ],
+  "author": "Michael Ruefenacht <michael.ruefenacht@joinbox.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/joinbox/loopback-nunjucks/issues"
+  },
+  "homepage": "https://github.com/joinbox/loopback-nunjucks#readme",
+  "devDependencies": {
+    "@joinbox/eslint-config-joinbox": "^1.0.8",
+    "chai": "^4.2.0",
+    "cheerio": "^1.0.0-rc.2",
+    "eslint": "^4.19.1",
+    "mocha": "^5.2.0",
+    "nyc": "^13.3.0"
+  },
+  "dependencies": {
+    "@joinbox/loopback-microservice": "^1.2.1",
```

Comment:
I think it doesn't matter as we use the package for all out microservices, but isn't this a dev dependency for the nunjuks package?